common/src/datetime.rs

```diff
@@ -14,23 +14,25 @@ use time::{OffsetDateTime, PrimitiveDateTime, UtcOffset};
 #[serde(rename_all = "lowercase")]
 pub enum DateTimePrecision {
     /// Second precision.
+    #[serde(alias = "secs")]
+    #[serde(alias = "seconds")]
     #[default]
     Second,
     /// Millisecond precision.
+    #[serde(alias = "millis")]
+    #[serde(alias = "millisecs")]
+    #[serde(alias = "milliseconds")]
```

Comment:
Is there a reason for all the aliases? I'd prefer just one variant

Reply:
it is something we put in quickwit config. I thought it could be nice to be a tad lenient on those.